test/index.js

```diff
@@ -0,0 +1,32 @@
+"use strict";
+
+const babelPresetEnv = require("../lib/index.js");
```

Comment:
NOTE: the tests here rely on built code. Happy to switch it up and run tests through babel or something if you'd prefer.